### PR TITLE
Adding Acceptance tests for the Doing Business with Us pages

### PR DESCRIPTION
**Pull request by sebworks**
Acceptance tests for the `Doing Business with Us pages` pages

## Changes
 - Added Acceptance tests for the `Doing Business with Us pages` pages. 

## Testing
-  Run `gulp test:acceptance`

## Review
@anselmbradford 
@jimmynotjim 
@KimberlyMunoz

### CODE REVIEW

File: test/browser_tests/page_objects/page_dbwu-upcoming-procurement-needs.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  this.procurementNeedsExternalLink =
+  element( by.css( '[data-qa-hook="main-summary"] + .short-desc a' ) );
+
+  this.procurementNeedsTable =
+  element( by.css( '.content_main .simple-table' ) );
+
+  this.moreInfoSection =
+  element.all( by.css( '[data-qa-hook="more-info"]' ) );
+
+  this.moreInfoSectionTitles = this.moreInfoSection.all( by.css( 'h1' ) );
```
Comment (KimberlyMunoz) [inline test/browser_tests/page_objects/page_dbwu-upcoming-procurement-needs.js]: This markup will change. There's only one h1 on the page now and that's the page title. Maybe a data-hook instead?

Comment (KimberlyMunoz) [inline test/browser_tests/page_objects/page_dbwu-upcoming-procurement-needs.js]: Actually, nevermind, you use a data-hook above. This is :ok_hand:, the test can change when the markup changes.

**Comment by KimberlyMunoz** [comment]
:+1: tests pass!

**Comment by anselmbradford** [comment]
:+1: test pass locally. I get the error `Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.` on Windows on Sauce, but looks like a test timeout error, so not an issue with this PR.

File: test/browser_tests/page_objects/page_dbwu-small-businesses.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  this.get = function() {
+    browser.get( 'doing-business-with-us/small-businesses/' );
+  };
+
+  this.pageTitle = browser.getTitle;
+
+  this.sideNav = element( by.css( '.nav-secondary' ) );
+
+  this.mainTitle = element( by.css( '[data-qa-hook="main-title"]' ) );
```
Comment (jimmynotjim) [inline test/browser_tests/page_objects/page_dbwu-small-businesses.js]: Same as other tests, update to use `_getQAelement` where possible.

**Comment by jimmynotjim** [comment]
Same question as other tests, should the mappings be updated for [this comment](https://github.com/cfpb/cfgov-refresh/pull/1005#discussion_r40157279)?

File: test/browser_tests/page_objects/page_dbwu-small-businesses.js (hunk 2 above)
Comment (sebworks) [inline test/browser_tests/page_objects/page_dbwu-small-businesses.js]: fixed in https://github.com/cfpb/cfgov-refresh/commit/aa3d436b18af4069fff661b4b51d0fa772a64846

**Comment by jimmynotjim** [comment]
:+1:

**Pull request merged**